ResumeParseAndNormalize.md

```diff
@@ -348,6 +349,10 @@ Response Structure
 }
 ```
 
+PII Scrubbing
+-----------
+The service makes a best attempt at scrubbing Personally Identifiable Information i.e. (Social Security Numbers and Driver's License Numbers). PII scrubbing logic is applied to the raw_resume_text, clean_resume_text, and resume_html fields only. Parsed data fields (such as the phone number fields) will therefore be unaffected by false-positive PII removals. Anything found to resemble a SSN or Driver's License Number in the resume text will be replaced with either: "***Social Security Number Removed***" or "***Driver's License Number Removed***". In addition the service by default scrubs information resembling a date of birth. To see how to disable this feature please review the return_date_of_birth field in the [Request structure](#request-structure).
```

Comment:
The "i.e." should be part of the parenthetical. (Read the sentence if you remove the parenthetical to see if it still makes sense)

You should surround field names with `backticks` for the monospaced formatting (tells the reader that it's something pertaining to code)

Move the "Anything found..." sentence after the "PII scrubbing..." sentence and change "in the resume text" to "in these fields"

"In addition" should be followed by a comma. I'd also revise this sentence slightly: "In addition, the service attempts to scrub date of birth from the same set of fields." I'd also start a new paragraph for this part since it's kind of a different idea.

"in the [Request structure]." --> "in the [Request structure] section."